src/styles/index.js

```diff
@@ -3,6 +3,7 @@ export const CONTROL_SIZE = 44;
 export const NAVBAR_SIZE = 58;
 export const SPACING = 16;
 export const HALF_SPACING = 8;
+export const QUARTER_SPACING = 8;
```

Comment:
Just wondering, if `HALF_SPACING` is `8`, then `QUARTER_SPACING` will be right? 🤔

Reply:
Oops, that was a bad mistake. Will fix right away!